src/environmentd/tests/sql.rs

```diff
@@ -1298,11 +1298,18 @@ fn test_utilization_hold() {
 
     let mut client = server.connect(postgres::NoTls).unwrap();
 
+    // Use the indexes
+    client.execute("SET cluster=mz_introspection", &[]).unwrap();
+
```

Comment:
I think that this no longer tests that the tables retention window works, am I reading that right? Maybe we should execute the query twice, once in the `mz_introspection` cluster using the index and once out of that cluster using the table.